src/commands/functions/functions-create.js

```diff
@@ -258,8 +302,17 @@ const ensureFunctionDirExists = async function (command) {
  * @param {import('commander').OptionValues} options
  * @param {string} argumentName
  * @param {string} functionsDir
+ * @param {boolean} isEdgeFunc
  */
-const downloadFromURL = async function (command, options, argumentName, functionsDir) {
+// eslint-disable-next-line max-params
+const downloadFromURL = async function (command, options, argumentName, functionsDir, isEdgeFunc) {
```

Comment:
Rather than disabling the `max-params` rule, you could look into using named parameters.

```suggestion
const downloadFromURL = async function ({ command, options, argumentName, functionsDir, isEdgeFunc }) {
```

Reply:
Makes sense. Will do!